packages/rest/test/unit/coercion/utils.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  OperationObject,
+  ParameterObject,
+  SchemaObject,
+} from '@loopback/openapi-v3-types';
+
+import {
+  ShotRequestOptions,
+  expect,
+  stubExpressContext,
+} from '@loopback/testlab';
+
+import {
+  PathParameterValues,
+  Request,
+  Route,
+  createResolvedRoute,
+  parseOperationArgs,
+  ResolvedRoute,
+} from '../../..';
+
+export function givenOperationWithParameters(params?: ParameterObject[]) {
+  return <OperationObject>{
+    'x-operation-name': 'testOp',
+    parameters: params,
+    responses: {},
+  };
+}
+
+export function givenRequest(options?: ShotRequestOptions): Request {
+  return stubExpressContext(options).request;
+}
+
+export function givenResolvedRoute(
+  spec: OperationObject,
+  pathParams: PathParameterValues = {},
+): ResolvedRoute {
+  const route = new Route('get', '/', spec, () => {});
+  return createResolvedRoute(route, pathParams);
+}
+
+export async function testCoercion<T>(
+  schemaSpec: SchemaObject,
+  valueFromReq: string,
+  expectedResult: T,
+  caller: string,
+) {
+  try {
+    const req = givenRequest();
+    const spec = givenOperationWithParameters([
+      {
+        name: 'aparameter',
+        in: 'path',
```

Comment:
Is it ok to use a hard-coded value `in: 'path'` for all tests?

Reply:
@bajtos good point, I can create sugar functions based on the parameter position.